rules/type-declaration/src/Rector/Identical/FlipTypeControlToUseExclusiveTypeRector.php

```diff
@@ -125,7 +125,7 @@ private function processConvertToExclusiveType(array $types, Expr $expr, PhpDocI
         /** @var VarTagValueNode $tagValueNode */
         $tagValueNode = $phpDocInfo->getVarTagValueNode();
         $phpDocInfo->removeTagValueNodeFromNode($tagValueNode);
-        return new BooleanNot(new Instanceof_($expr, new FullyQualified($type)));
+        return new BooleanNot(new Instanceof_($expr, new Name($type)));
```

Comment:
This should work with FQN, as we use it everywhere in Rector by convention.

The name resolution of `Closure` must be fixed above, probably in `$type`.